modules/op-configuring-buildah-to-use-build-user.adoc

```diff
@@ -6,22 +6,26 @@
 [id="configuring-builah-to-use-build-user_{context}"]
 = Configuring Buildah to use `build` user
 
-You can define a Buildah task to use the `build` user with user id `1000`. 
+You can define a Buildah task to use the `build` user with user id `1000`.
+
+.Prerequisites
+
+* You must install the link:https://mikefarah.gitbook.io/yq/[`yq` tool].
```

Comment:
@mramendi we cannot add links to GH repos that are not maintained by our team.

Reply:
@Preeticp please advice a course of action. I checked and yq does not seem to be available as a package in the Fedora repository. If I just write the name of the tool without pointing to a way to get it, the usability suffers.